src/ensembl/utils/database/dbconnection.py

```diff
@@ -74,6 +75,17 @@ def load_metadata(self) -> None:
         self._metadata = sqlalchemy.MetaData()
         self._metadata.reflect(bind=self._engine)
 
+    def create_all_tables(self, metadata: MetaData) -> None:
+        """Create the tables from the metadata and set the metadata."""
+        self._metadata = metadata
+        metadata.create_all(self._engine)
```

Comment:
If a user does that then I'd say it's on them to use `load_metadata()` to ensure the metadata is up to date...

We should not load the metadata by default because it has a cost. I guess I should add a warning to reload after if the database is not new/may have other tables.

Reply:
Yes, I'm tempted to say that at least a clear warning + updated docstring needs to be in place.